index.html

```diff
@@ -79,61 +91,70 @@ <h5>I'm a software developer! I can help you build a product ,feature or website
-                            <li>Ruby</li>
-                            <li>Frontend</li>
-                        </ul>
-                        <div class="action">
-                            <button class="seeSource">See Source</button>
-                            <button class="seeLive">See Live</button>
-                        </div>
-                    </div>
-
-                    <div class="left-block">
-                        <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry.
-                            Lorem Ipsum has been the industry's standard dummy text ever since the 1500s,
-                            when an unknown printer took a galley of type and scrambled it 1960s with
-                            the releaLorem Ipsum is simply dummy text of the printing and typesetting industry.
-                            Lorem Ipsum has been the industry's standard dummy text ever since the 1500s,
-                            when an unknown printer took a galley of type and scrambled it 1960s with the
-                            releorem Ipsum is simply dummy text of the printing and typesetting industry.
-                            Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the
-                            releawn printer took a galley of type and scrambled it 1960s with the releaLorem
-                            Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum
-                            has been the industrys standard dummy text ever since the 1500s, when an unknown
-                            printer took a galley of type and scrambled it 1960s with the relea</p>
+    
+                 <div class="frame2">
+                    <ul class="card-frame">
+                        <li>CANOPY</li>
+                        <li class="no-count">Back End Dev</li>
+                        <li class="no-count">2015</li>
+                    </ul>
+                 </div>
+                </div>
+    
+                <div class="img-popup">
+                    <img class="popup-img modal-content">
+                </div>
+                
+                <div class="right-block modal-body">
+                  <p class="popups-project-p-mobile"></p>
+                  <ul class="card-tags2">
+                      <li class="card-tagsLi">HTML</li>
+                      <li class="card-tagsLi">css</li>
+                      <li class="card-tagsLi">javascript</li>
+                  </ul>
+                  
+                  <ul class="card-tags1">
+                      <li class="card-tagsLi1">Github</li>
+                      <li class="card-tagsLi1">Ruby</li>
+                      <li class="card-tagsLi1">Frontend</li>
+                  </ul>
+                  <hr class="divider2">
+                  <div class="action">
+                      <button class="seeSource see"><p>See Source</p>
```

Comment:
- [ ] The modal pop up for mobile version is quite good, but there is a problem with the styling of the text `SEE SOURCE` button as you can see in the screenshot below. Also centralise the `SEE LIVE` text. 
- [ ] Kindly give your project description a left margin and justify the text as shown on the screenshot of the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=ras6Y0lSfGI2Obse-0) design.
- Your design
![image](https://user-images.githubusercontent.com/62849872/210357481-85ac8343-e59f-4332-863b-b4888b72ac04.png)
- Figma Design
![image](https://user-images.githubusercontent.com/62849872/210358126-0c9d8a96-3acd-465d-afc8-52cb3319d69a.png)